client/src/components/game/lobby/Lobby.css

```diff
@@ -0,0 +1,21 @@
+.lobby-container {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  gap: 0.5rem;
+}
+
+.lobby-avatar {
+  width: 2.2rem;
+  border: 1.5px solid;
+  border-radius: 50%;
+  vertical-align: top;
```

Comment:
Ran into this weird [issue](https://stackoverflow.com/questions/19212352/div-height-based-on-child-image-height-adds-few-extra-pixels-at-the-bottom)